storage/filesystem/index.go

```diff
@@ -1,3 +1,4 @@
+//nolint:revive // interface methods don't need individual comments
```

Comment:
Can we disable this wholesale from the config instead?

Reply:
yes, should be cleaner now